### DISCUSSION TITLE
Not able to use the benefits of Github Student Pack after getting accepted

### DISCUSSION THREAD

**Discussion by TruonghocAI** [community/community#58467] [GitHub Education]
### Hi everyone,

Hello everyone, days of getting accepted.
I am attaching the screenshot too please look into it and solve my error.
thanks a lot!!!
I am not able to use my github student benefits after it's been 4 days of getting accepted.
I am attaching the screenshot too please look into it and solve my error.
thanks a lot!!!
<img width="960" alt="2023-06-19" src="https://github.com/community/community/assets/128415405/978f16d0-19d0-49cd-af42-61a22e017294">

**Comment by mrepol742** [comment]
Are you sure you cant use your github student benefits at all?
![image](https://github.com/community/community/assets/62317165/7455b208-0d8c-4f6f-a892-78f3b2d7c2d3)
You are already in Github Pro which is part of the student package.

Check this page:
https://education.github.com/pack/offers

**Comment by samus-aran** [comment]
Apologies for the delayed response from the team, we hope you were able to resolve your rejection reasons and receive a successful verdict on your application. I'll be closing this discussion due to lack of recent activity. If you still haven't received your Global Campus benefits please feel free to re-open and we'll be happy to assist!
